torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -139,6 +139,13 @@ void XLAGraphExecutor::DeviceContextArena::SaveGraphAsString(
   }
 }
 
+void XLAGraphExecutor::DeviceContextArena::SaveOutputShapes(
+    torch::lazy::hash_t hash, std::vector<xla::Shape> outptu_shapes) {
```

Comment:
outptu_shapes => output_shapes, and const std::vector<xla::Shape>& perhaps?

Reply:
will fix